dgfit/dustmodel.py

```diff
@@ -495,7 +499,9 @@ def save_results(self, filename, OD, size_dist_uncs=[0]):
         k1 = 0
         for component in self.components:
             col1 = fits.Column(name="SIZE", format="E", array=component.sizes)
-            col2 = fits.Column(name="DIST", format="E", array=component.size_dist)
+            col2 = fits.Column(
+                name="DIST", format="E", array=(component.size_dist / OD.avnhi)
```

Comment:
Does this need to be convert to per A(V)?  I would have thought that if the observed data was in per A(V), then the size distributions would naturally be in per A(V) already.